### PR TITLE
Apply patch from hyperboreanwolfgirl for Windows Server 2022

### PR DESCRIPTION
**Pull request by Matthew-Beckett**
This PR applies the patch created by hyperboreanwolfgirl to support Windows Server 2022

https://github.com/hyperboreanwolfgirl/py-kms/commit/4b6a408e4f1680d71524740499f8909ce651f48d.patch

### CODE REVIEW

File: py-kms/KmsDataBase.xml [hunk 1]
```diff
@@ -103,6 +103,10 @@
 			<Activate KmsItem="7ba0bf23-d0f5-4072-91d9-d55af5a481b6" />
 		</CsvlkItem>
 
+		<CsvlkItem DisplayName="Windows Server 2022" VlmcsdIndex="0" GroupId="206" MinKeyId="551000000" MaxKeyId="570999999" IniFileName="Windows" EPid="06401-00206-566-174993-03-1033-9600.0000-2802018" Id="ef6cfc9f-8c5d-44ac-9aad-de6a2ea0ae03" InvalidWinBuild="[0,1,2]">
```
Comment (Matthew-Beckett) [inline py-kms/KmsDataBase.xml:106]: I am concerned by the clashing values for `GroupId` and `EPid` I have not tested this change and I am unsure on the success of @hyperboreanwolfgirl in their attempt to get this working.

**Comment by Matthew-Beckett** [comment]
@simonmicro do not merge this yet, I have not tested this.

**Comment by xadammr** [comment]
* Activation works prior to having updated KmsDataBase.xml
Screenshot 2023-08-27 at 09 48 53

Tested by overloading the `KmsDataBase.xml` via Docker:
```
    volumes:
      - /srv/kms/KmsDataBase-9bdc0e4.xml:/home/py-kms/KmsDataBase.xml
```

Windows Server 2022 now appears in the GUI under Products... But the existing activations do not reflect the SkuID-- it remains showing the GUID. 

Perhaps that's a fault with the web interface, but regardless, this doesn't break activation.

**Comment by xadammr** [comment]
New activation works as well, but it's not capturing the correct name the SKU ID field:

Screenshot 2023-08-27 at 13 19 41

**Comment by BenPi-Ross** [comment]
I've downloaded this version of the "KmsDataBase.xml" ([KmsDataBase_mod.xml.txt](https://github.com/Py-KMS-Organization/py-kms/files/14441080/KmsDataBase_mod.xml.txt)) a few months ago and I remember modifying it but I don't remember exactly what I modified.

Anyway it works fine on my side with that file :
image

The only relevant difference that I found that isn't in hyperboreanwolfgirl patch is this line :
`    <WinBuilds>
        [...]
        <WinBuild BuildNumber="20348" DisplayName="Windows Server 2022" UseForEpid="true" PlatformId="3612" MayBeServer="true" UsesNDR64="true" MinDate="18/08/2021"/>
    </WinBuilds>`

**Comment by simonmicro** [comment]
Idk, I'll just go ahead and merge this one. It seems stale, but in another issue users mentioned, this works.

**Pull request merged**